app/assets/v2/js/base.js

```diff
@@ -403,10 +403,6 @@ const gitcoinUpdates = (force) => {
   });
 };
 
-if (document.contxt.github_handle) {
-  gitcoinUpdates();
-}
```

Comment:
^ can we update the `gitcoinUpdates` to remove the `force` argument + update it's invocation in `footer.html`

Reply:
We can do that.

Reply:
Removing `force` will not stop the popups. Hence I had to remove it entirely.

Reply:
I meant we remove this and update the function so that it no longer needs an argument `force`

Reply:
Without the force, the popup will still be there

Reply:
Ah it doesn't get invoked elsewhere ! worth removing the entire function itself?

Reply:
Noted.

Reply:
Its called from the footer as well as on login. I've removed the login call. Except you need me to remove the entire function and the link on footer

Reply:
@iamonuwa updated the code to show what i meant !
Removed unused code in the function